homeassistant/components/knx/sensor.py

```diff
@@ -294,11 +294,11 @@ class SensorConfig(EntityConfiguration, ABC):
     @classmethod
     def get_platform(cls) -> str:
         """Return the platform name for this configuration."""
-        return Platform.SENSOR.value
+        return str(Platform.SENSOR)
```

Comment:
`Platform.SENSOR` is a `StrEnum` - it should behave like a string wherever needed, doesn't it? (It can also be nice for typing purposes, but I didin't look up where `get_platform` is used now).